### PR TITLE
BAT-5475 - Use permanent redirects

### PR DESCRIPTION
**Pull request by olivierlefloch**

### CODE REVIEW

**Comment by asiebert** [comment]
…you can never settle for a single change on that project, hmm? x)

**Comment by olivierlefloch** [comment]
I'd rather batch stuff. Painful to have 15 tasks open at a time :)

File: django_short_urls/views.py [hunk 1]
```diff
@@ -101,21 +98,17 @@ def main(request, path):
         defaults={REF_PARAM_NAME: REF_PARAM_DEFAULT_VALUE}
     )
 
-    # FIXME: OPS-3885 - we're logging agressively to Sentry here because we don't understand some weird 404 errors on
-    # FIXME: workfor.us. We suspect they are being triggered by some pages being served directly as a proxy, perhaps.
-    if link.act_as_proxy:
-        logging.error('Page served through proxy: %s', str(link))
```
Comment (asiebert) [inline django_short_urls/views.py]: Can't seem to find any such recent errors on Sentry, is that where it's supposed to be? Was there a conclusion somewhere about this?

Comment (olivierlefloch) [inline django_short_urls/views.py]: There was no conclusion, but none of these errors appear anymore... And we have not had any end user reports of anything that could relate to this.

Comment (asiebert) [inline django_short_urls/views.py]: I guess that'll count as the conclusion!
:white_check_mark:

**Pull request merged**